src/NoTimeToWaitBundle/Resources/views/Default/nttw8.html.twig

```diff
@@ -55,7 +55,7 @@
 
 <h2>Travel Grants</h2>
 
-<p>We're currently working on budgeting for travel grants and will announce a travel grant application on this page and at <a href="https://twitter.com/nttwconf">our twitter account</a> when they're ready.</p>
+<p>This year we're able to offer 2 250€ travel grants. Thanks in particular to MEMORIAV for sponsoring these travel grants. To apply please complete this <a href="https://bit.ly/nttw8travelgrant">travel grant application</a>. We'll start reviewing and awarding applications on August 2, 2024.</p>
```

Comment:
"2 250€" --> "2x 250 €" (space is a thousands separator in Europe, I read 2250, and units are separated from numbers by a space in Eurozone :-p)